std/builtin/base.neva

```diff
@@ -55,5 +55,5 @@ pub component Eq<T int |float | string>(a T, b T) (then T, else EqElse<T>)
 #extern(list list_len, map map_len)
 pub component Len<T list<any> | map<any>>(data T) (res int)
 
-
-
+#extern(get_index)
```

Comment:
Let's use the same naming `#extern(index)` or maybe `list_index` just to be consistent with the rest of the stdlib.

Proper naming is something we still need to figure out but there is no named like that. Probably more idiomatic naming would be `indexer` but I suggest stick with `index` for now.